packages/components-react/src/styles/StylesGridExample.tsx

```diff
@@ -1,531 +1,5 @@
-            Some label
-          </PButton>
-        </BasicContentOneThirdFollow>
-        <BasicContentOneThirdFollow>
-          <Info>
-            <b>Basic Third</b> for Content Tiles
-          </Info>
-          <Heading>Heading in Tile</Heading>
-          <PButton style={{ marginTop: spacingFluidMedium }} variant="secondary">
-            Some label
-          </PButton>
-        </BasicContentOneThirdFollow>
-        <BasicContentTwoThirdsLeft>
-          <Info>
-            <b>Basic Two Thirds</b> for Content Tiles
-          </Info>
-        </BasicContentTwoThirdsLeft>
-        <BasicContentOneThirdRight>
-          <Info>
-            <b>Basic One Third</b> for Content Tiles
-          </Info>
-        </BasicContentOneThirdRight>
-        <BasicContentCustomLeft>
-          <Info>
-            <b>Custom (desktop: column 3-7)</b> for Content
-          </Info>
-        </BasicContentCustomLeft>
-        <BasicContentCustomRight>
-          <Info>
-            <b>Custom (desktop: column 9-14)</b> for Content
-          </Info>
-        </BasicContentCustomRight>
-      </BasicContentGrid>
-      <NarrowContentGrid>
-        <NarrowContent>
-          <Info style={{ marginLeft: spacingStaticSmall, marginTop: spacingStaticSmall }}>
-            <b>Narrow</b> for small Components and Content
-          </Info>
-          <PAccordion heading="Some Heading" tag="h3" open={isAccordion1Open} onUpdate={onAccordion1Update}>
-            <Text>
-              Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Donec quam
-              felis, ultricies nec, pellentesque eu. Aenean massa.
-            </Text>
-          </PAccordion>
-          <PAccordion heading="Some Heading" tag="h3" open={isAccordion2Open} onUpdate={onAccordion2Update}>
-            <Text>
-              Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Donec quam
-              felis, ultricies nec, pellentesque eu. Aenean massa.
-            </Text>
-          </PAccordion>
-        </NarrowContent>
-        <NarrowContentHalfLeft>
-          <Info>
-            <b>Narrow</b> Half for small Content Tiles
-          </Info>
-          <Heading>Experience</Heading>
-          <Text>
-            Goosebumps, adrenaline: experience the fascination of sports cars - with all different facets and according
-            to your wishes.
-          </Text>
-        </NarrowContentHalfLeft>
-        <NarrowContentHalfRight>
-          <Info>
-            <b>Narrow</b> Half for small Content Tiles
-          </Info>
-          <Heading>Experience</Heading>
-          <Text>
-            Goosebumps, adrenaline: experience the fascination of sports cars - with all different facets and according
-            to your wishes.
-          </Text>
-        </NarrowContentHalfRight>
-      </NarrowContentGrid>
-    </>
-  );
+  return <GridLayout gridVisualizer={true} />;
```

Comment:
Let't stay consistent with the prop name and perhaps give it a more boolean name?